src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CborReaderTests.Array.cs

```diff
@@ -77,10 +77,10 @@ public static void ReadArray_NonCanonicalLengths_SupportedConformanceLevel_Shoul
         }
 
         [Theory]
-        [InlineData(CborConformanceLevel.Rfc7049Canonical, "9800")]
-        [InlineData(CborConformanceLevel.Rfc7049Canonical, "990000")]
-        [InlineData(CborConformanceLevel.Rfc7049Canonical, "9a00000000")]
-        [InlineData(CborConformanceLevel.Rfc7049Canonical, "9b0000000000000000")]
+        [InlineData(CborConformanceLevel.Canonical, "9800")]
```

Comment:
I liked the previous format better

Reply:
The RFC prefix? Given it's just the spec for CBOR itself, I found it a bit redundant. Lax and strict modes are also defined in the same RFC. The XML doc should have all the details regardless.

Reply:
@bartonjs thoughts?

Reply:
If we didn't know about the CTAP2 canonical conformance level I think we'd have just called it `Canonical`, and it's .. the canonical canonical form? :smile:

I'd take it in the short form, and be prepared to discuss the longer name.  Since either the RFC number or `Cbor` are really redundant with the type name, I think the short form (`Canonical`) is OK; but I can definitely see arguments for "there are two things named Canonical, can we better disambiguate them?".